pkg/build/gobuild.go

```diff
@@ -349,11 +371,30 @@ func spdx(version string) sbomber {
 	}
 }
 
+func writeSBOM(sbom []byte, appPath, dir, ext string) error {
+	if dir != "" {
+		appDir := path.Join(dir, appPath)
+		if _, err := os.Stat(appDir); errors.Is(err, os.ErrNotExist) {
```

Comment:
I think we can just always `MkdirAll`, which will exit successfully if it already exists.